### PR TITLE
rename AggregateLevel and ExpandLevel

### PR DESCRIPTION
**Pull request by luotao1**
rename AggregateLevel and ExpandLevel based on https://github.com/PaddlePaddle/Paddle/pull/2200#pullrequestreview-39012883

### CODE REVIEW

File: python/paddle/trainer_config_helpers/layers.py [hunk 1]
```diff
@@ -235,16 +235,16 @@ class AggregateLevel(object):
 
     Accordingly, AggregateLevel supports two modes:
 
-    - :code:`AggregateLevel.EACH_TIMESTEP` means the aggregation acts on each
+    - :code:`AggregateLevel.TO_NO_SEQUENCE` means the aggregation acts on each
```
Comment (emailweixu) [inline python/paddle/trainer_config_helpers/layers.py]: NO_SEQUENCE => NON_SEQUENCE
We can still keep the original names for compatibility.

Comment (luotao1) [inline python/paddle/trainer_config_helpers/layers.py]: @emailweixu If NO_SEQUENCE => NON_SEQUENCE, how about SequenceType.NO_SEQUENCE? Shall we rename the SequenceType at the same time ?

Comment (emailweixu) [inline python/paddle/trainer_config_helpers/layers.py]: 那就还时no_sequence吧

Comment (luotao1) [inline python/paddle/trainer_config_helpers/layers.py]: @emailweixu fix the travis-ci, please review again, thx!

Comment (emailweixu) [inline python/paddle/trainer_config_helpers/layers.py]: 以前的EACH_TIMESTEP等还需要保留一下吧，否则已有的配置会出错。

Comment (luotao1) [inline python/paddle/trainer_config_helpers/layers.py]: @emailweixu 已添加EACH_TIMESTEP等以前的用法。

**Pull request merged**